src/fsharp/vs/IncrementalBuild.fs

```diff
@@ -1537,7 +1532,6 @@ type IncrementalBuilder(tcGlobals,frameworkTcImports, nonFrameworkAssemblyInputs
     member __.FileChecked = fileChecked.Publish
     member __.ProjectChecked = projectChecked.Publish
     member __.ImportedCcusInvalidated = importsInvalidated.Publish
-    member __.AllDependenciesDeprecated = allDependencies 
```

Comment:
Is there a specific reason to remove this as part of this PR?  Just wondering, thanks

Reply:
No, not really, I've reverted it.